src/main/java/caculator/domain/CalculatorType.java

```diff
@@ -0,0 +1,46 @@
+package caculator.domain;
+
+import java.util.function.BiFunction;
+
+public enum CalculatorType {
+    PLUS("+", (num1, num2) -> num1 + num2),
+    MINUS("-", (num1, num2) -> num1 - num2),
+    MULTIPLY("*", (num1, num2) -> num1 * num2),
+    DIVIDE("/", (num1, num2) -> num1 / num2);
+
+    public static final int ZERO_NUMBER = 0;
+    private String type;
+    private BiFunction<Integer, Integer, Integer> expression;
```

Comment:
```suggestion
    private BinaryOperator<Integer> expression;
```
파라미터와 결과값이 같은 경우 BinaryOperator 형태도 가능합니다!